### ISSUE TITLE
tfrecords file???I use these code to make train.tfrecords, it is not work. why???

### ISSUE THREAD

**Issue by taoyunuo** [vanhuyz/CycleGAN-TensorFlow#43]
import tensorflow as tf
import os.path
import matplotlib.image as mpimg 
from PIL import Image 
SAVE_PATH = "C:\\CycleGAN-TensorFlow-master\\datasetB_new.tfrecords"
def _int64_feature(value):  
    return tf.train.Feature(int64_list=tf.train.Int64List(value=[value]))  
def _bytes_feature(value):  
    return tf.train.Feature(bytes_list=tf.train.BytesList(value=[value]))  
  
def load_data(datafile, width, high, method=0, save=False):  
    train_list = open(datafile,'r')  
    writer = tf.python_io.TFRecordWriter(SAVE_PATH)  
    with tf.Session() as sess:
        label=0
        for line in train_list:  
            tmp = line.strip().split(' ')  
            img_path = tmp[0]  
            image = tf.gfile.FastGFile(img_path, 'rb').read()
            image = tf.image.decode_jpeg(image)  
            image = tf.image.convert_image_dtype(image, dtype=tf.float32)  
            image = sess.run(image)  
            image_raw = image.tostring()   
            example = tf.train.Example(features=tf.train.Features(feature={  
                'image_raw': _bytes_feature(image_raw),  
                'label': _int64_feature(label),  
                }))  
            label = label+1 
            writer.write(example.SerializeToString())  
  
    writer.close()  
load_data('C:\\CycleGAN-TensorFlow-master\\samples\\monet2photo\\monet2photo\\trainB.txt', 256,256)

**Comment by vanhuyz** [comment]
Hi @taoyunuo 
Please post your error log!

**Comment by taoyunuo** [comment]
thanks，I have solved this problem@vanhuyz
